model/karma.py

```diff
@@ -25,7 +25,7 @@ def _get_current_net_karma(**kwargs) -> list:
                                    "awarded": "$awarded"}}
         match = {"$match": {'awarded': {'$gt': _get_cut_off_date()}}}
         grouping = {"$group": {"_id": "$recipient", "net_karma": {"$sum": "$net_karma"}}}
-        sort = {"$sort": {"net_karma": -1}}
+        sort = {"$sort": {"net_karma": -1, "_id": 1}}

```

Comment:
I'm reasonably sure as soon as we add secondary sorts, the primary sort goes bye-bye on Windows, which is why there isn't one :-/.

Reply:
Sorry that wasn't supposed to be in this request anyway...